frontend/pages/hosts/ManageHostsPage/ManageHostsPage.jsx

```diff
@@ -341,7 +341,7 @@ export class ManageHostsPage extends PureComponent {
         onExit={toggleDeleteHostModal(null)}
         className={`${baseClass}__modal`}
       >
-        <p>This action will delete the host from your Fleet instance.</p>
+        <p>This action will delete the host <strong>${selectedHost.hostname}</strong> from your Fleet instance.</p>
```

Comment:
Is the `$` intentional?

```suggestion
        <p>This action will delete the host <strong>{selectedHost.hostname}</strong> from your Fleet instance.</p>
```

Reply:
Apologies, I don't have much experience with frontend languages. It was my understanding that it was needed when inserting the variable to a string, but now that I'm looking at this again this isn't actually in a string block, so it isn't needed.